service/matching/db.go

```diff
@@ -745,7 +749,13 @@ func (db *taskQueueDB) emitBacklogGaugesLocked() {
 		}
 	}
 
-	metrics.ApproximateBacklogCount.With(db.metricsHandler).Record(float64(approximateBacklogCount))
+	for _, v := range count {
+		priStr := ""
+		if v > 0 {
+			priStr = strconv.FormatInt(v, 10)
+		}
```

Comment:
I think it's about time to add a helper for this, maybe in the metrics package like other tag wrappers?

Not blocking this PR though